### PR TITLE
[script.episodehunter] 2.0.4

### PR DESCRIPTION
**Pull request by tjoskar**
### Description:
Update addon for episodehunter. 

* Python 3 compatible
* Remove offline database 
* Remove support for movie and show sync 

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by tjoskar** [comment]
Hi,
I'm not sure why travis is failing. It looks like (from the error message) that `kodi-addon-checker` is called without the `branch` argument which is mandatory? Or I'm I missing something 🙂?

**Comment by Razzeee** [comment]
Yes, correct. My bad. @tjoskar if you can, please rebase your branch and the error should be gone :)

If you can't rebase, let us know and we will figure somthing out.

**Comment by tjoskar** [comment]
Thanks @Razzeee!

**Comment by enen92** [comment]
@tjoskar script.episodehunter is also in other repositories such as gotham and helix. I assume you want to keep a different version for Krypton and above, is the assumption correct?
If so there is no problem with keeping the other 2 versions (as long as they still work). If they are broken can you please mark them as broken?

**Comment by tjoskar** [comment]
@enen92, The older version works on both Gotham and Helix and yes that was my intention (two have two different versions) but now when I think about it; I should update the addon for Gotham, Helix, Isengard and Jarvis as well.

Questions: 
1. Should I close this PR and create a new one for Gotham (and change the `xbmc.python` version to 2.14.0) and it will automatically be accessible for Gotham to Krypton?
2. The language resource has moved from `resources/language/English/strings.po` to `resources/language/resource.language.en_gb/strings.po` in Krypton(?). Is is safe to keep the language file in `resources/language/English/strings.po` or should I create a copy?

File: script.episodehunter/addon.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<addon id="script.episodehunter" version="2.0.4" name="EpisodeHunter" provider-name="Episodehunter.tv">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.service" name="episodeHunter.service" library="addon.py"/>
+    <extension point="xbmc.addon.metadata">
+        <summary>EpisodeHunter</summary>
```
Comment (enen92) [inline script.episodehunter/addon.xml]: `lang="en_GB"` please

File: script.episodehunter/addon.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <description>
```
Comment (enen92) [inline script.episodehunter/addon.xml]: `lang="en_GB"` please

File: script.episodehunter/addon.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Episodehunter keeps automatically a record of what you are watching and can there by keep a comprehensive history of all TV shows and movies you have watched and give tips on what to see next.
+You are required to have an EpisodeHunter account for this plugin.
+Create an account on episodehunter.tv
+        </description>
+        <news>v2.0.4 (2018-06-03)
+- Remove movie and shows sync
+- Remove offline database</news>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <language>en</language>
```
Comment (enen92) [inline script.episodehunter/addon.xml]: This is only for content. Does the addon provide any content? If not leave blank or remove

**Comment by tjoskar** [comment]
@enen92, Yeah, I think you are right. I let this version only be for Krypton. Thanks for your comments!

**Comment by enen92** [comment]
Thanks for the submission, merged

**Pull request merged**